code/modules/telesci/bcrystal.dm

```diff
@@ -0,0 +1,71 @@
+// Bluespace crystals, used in telescience and when crushed it will blink you to a random turf.
+
+/obj/item/bluespace_crystal
+	name = "bluespace crystal"
+	desc = "A glowing bluespace crystal, not much is known about how they work. It looks very delicate."
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "bluespace_crystal"
+	w_class = 1
+	origin_tech = list(TECH_BLUESPACE = 4, TECH_MATERIAL = 3)
+	var/blink_range = 8 // The teleport range when crushed/thrown at someone.
+
+
+/obj/item/bluespace_crystal/New()
+	..()
+	pixel_x = rand(-5, 5)
+	pixel_y = rand(-5, 5)
+//	create_reagents(10)
+//	reagents.add_reagent("bluespace_dust", blink_range)
+
+/obj/item/bluespace_crystal/attack_self(mob/user)
+	user.visible_message("<span class='warning'>[user] crushes [src]!</span>", "<span class='danger'>You crush [src]!</span>")
+	var/datum/effect/effect/system/spark_spread/sparks = new /datum/effect/effect/system/spark_spread()
+	sparks.set_up(5, 0, get_turf(user))
+	sparks.start()
+	playsound(src.loc, "sparks", 50, 1)
+	playsound(src.loc, 'sound/effects/phasein.ogg', 25, 1)
+	blink_mob(user)
+	user.unEquip(src)
+	qdel(src)
+
+/obj/item/bluespace_crystal/proc/blink_mob(mob/living/L)
+	var/turf/T = get_random_turf_in_range(L, blink_range, 1)
+	L.forceMove(T)
+	var/datum/effect/effect/system/spark_spread/sparks = new /datum/effect/effect/system/spark_spread()
+	sparks.set_up(5, 0, T)
+	sparks.start()
+
+/obj/item/bluespace_crystal/throw_impact(atom/hit_atom)
+	if(!..()) // not caught in mid-air
+		visible_message("<span class='notice'>[src] fizzles and disappears upon impact!</span>")
```

Comment:
use SPAN_NOTICE() , search for other <span class='notice'> in your code and fix that too